### DISCUSSION TITLE
det模型训练之后和infer效果不一致

### DISCUSSION THREAD

**Discussion by 1443244362** [PaddlePaddle/PaddleOCR#13536] [Q&A] [answered]
### Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.

- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.

- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### Bug

![1720875794 125684](https://github.com/user-attachments/assets/176d6b12-73b0-4416-8e33-2bd7e574d131)
![1](https://github.com/user-attachments/assets/90244817-c118-4437-843a-b91a73b8dd01)
这是我两次检测的结果 我的训练模型检测效果十分好 转换成推理模型之后效果很差 我查阅了相关issue发现把predict_det.py的"limit_side_len": args.det_limit_side_len注释掉就可以了 但是我需要在python代码中使用 比如这样 ocr = PaddleOCR(lang="ch",det_model_dir="ai/models/det_db_inference", show_log=False) 我查了文档发现这里没有办法设置limit_side_len 所以效果就像我上传的图片一样非常差 我应该怎么办呢

### Environment

系统 win10
内存 32g 
显卡 4060ti 16g
paddle版本 2.6.1
cuda 12.3
cudnn 8.9

### Minimal Reproducible Example
```python
import cv2
import time
import sys
import os
import numpy as np
parent_folder_path = os.path.abspath(os.path.join(
    os.path.dirname(__file__), '..'))
sys.path.append(parent_folder_path)
from paddleocr import PaddleOCR
from libs.utils import read_image,write_image
ocr = PaddleOCR(lang="ch",det_model_dir="ai/models/det_db_inference", show_log=False)
if len(sys.argv) >1:
    img = read_image(sys.argv[1])
else:
    from libs.appium_driver import driver
    img = driver.getScreenshotForCv2()

start = time.time()
result = ocr.ocr(img,cls=False,rec=False)
print(f'time: {time.time()-start}')
for idx in range(len(result)):
    res = result[idx]
    if res is None:
        continue
    for line in res:
        print(line)
        points = np.array(line, dtype=np.int32)
        cv2.polylines(img, [points], isClosed=True, color=(0, 0, 255), thickness=1)
    # cv2.imshow('Image with Annotations', img)
    # cv2.waitKey(0)
    # cv2.destroyAllWindows()
cv2.imwrite('1.png',img)
```
这是我测试的代码 效果就是第二张图片

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by GreatV** [comment, marked as answer]
试试传参加上 `det_limit_side_len=736`

**Reply by WelcomeToWonderland** [reply]
ocr = PaddleOCR(use_angle_cls=True, rec = False, det_model_dir=det_model_dir, det_limit_side_len=3000)
传参，是指在创建PaddleOCR实例是，在后面添加det_limit_side_len=3000吗？
我这样尝试之后，效果没有发生变化

**Comment by GreatV** [comment]
参考文档：https://paddlepaddle.github.io/PaddleOCR/ppocr/model_train/detection.html#5-faq

**Comment by 1443244362** [comment]
> det_limit_side_len=736

厉害了 问题解决 但是736确实不够 我直接设置了3000 有没有什么参数可以直接让模型不限制这个呢
另外这个传参到底在文档的哪里能看到呢 我真找不到啊 能给个链接吗

**Reply by WelcomeToWonderland** [reply]
ocr = PaddleOCR(use_angle_cls=True, rec = False, det_model_dir=det_model_dir, det_limit_side_len=3000)
你好，请问“传参”操作，是这样实现吗？
我试过之后没有效果改变

**Comment by UserWangZz** [comment]
参考链接：https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/inference_args.md
https://paddlepaddle.github.io/PaddleOCR/ppocr/blog/inference_args.html

**Reply by WelcomeToWonderland** [reply]
参考链接已经过期了，麻烦大佬可以更新一下吗
